lib/property/changeAuthor.ts

```diff
@@ -0,0 +1,21 @@
+import { Contract } from 'web3-eth-contract/types'
+import Web3 from 'web3'
+import { execute } from '../utils/execute'
+import { T } from 'ramda'
+
+export type CreateChangeAuthorCaller = (
+	contract: Contract,
+	client: Web3
+) => (nextOwther: string) => Promise<boolean>
+
+export const createChangeAuthorCaller: CreateChangeAuthorCaller = (
+	contract: Contract,
+	client: Web3
+) => async (nextOwther: string) =>
```

Comment:
@Akira-Taniguchi  Is this typo? `nextOwther`

Reply:
yes, this is typo.
I will update it now.

Reply:
I fix it now. @aggre